addons/sourcemod/scripting/surftimer/hooks.sp

```diff
@@ -1457,23 +1457,27 @@ public Action ResetOneJump(Handle timer, any userid)
 public Action DelayedVelocityCap(Handle timer, Handle pack)
 {
 	ResetPack(pack);
-	int client = ReadPackCell(pack);
+	int client = GetClientOfUserId(ReadPackCell(pack));
 	float speedCap = ReadPackFloat(pack);
+	delete pack;
```

Comment:
DelayedVelocityCap is a timer callback which is called by [`CreateDataTimer`](https://sm.alliedmods.net/new-api/timers/CreateDataTimer) so we don't need to close the datapack which comes as it's parameter.

Before making changes that close datapacks in timer callback parameters I recommend to always check if it's called by [`CreateTimer`](https://sm.alliedmods.net/new-api/timers/CreateTimer) or [`CreateDataTimer`](https://sm.alliedmods.net/new-api/timers/CreateDataTimer).

Reply:
Yeah I remember... 🤦‍♂️
https://github.com/surftimer/Surftimer-olokos/pull/103#pullrequestreview-407890736

I changed that to CreateTimer now.